main.py

```diff
@@ -0,0 +1,63 @@
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import schemas
+import crud
+from db.database import SessionLocal
+
+app = FastAPI()
+
+
+def get_db() -> Session:
+    db = SessionLocal()
+    try:
+        yield db
+    finally:
+        db.close()
+
+
+@app.get("/authors/", response_model=list[schemas.AuthorList])
+def read_authors(
+    skip: int = 0, limit: int = 10, db: Session = Depends(get_db)
+):
+    authors = crud.get_all_authors(db, skip, limit)
+    return authors
+
+
+@app.post("/authors/", response_model=schemas.AuthorCreate)
+def create_author(author: schemas.AuthorCreate, db: Session = Depends(get_db)):
+    return crud.create_author(db=db, author=author)
+
+
+@app.get("/authors/{author_id}", response_model=schemas.AuthorList)
```

Comment:
```suggestion
@app.get("/authors/{author_id}/", response_model=schemas.AuthorList)
```
the same in other similar cases